### ISSUE TITLE
templates/ondemand.yml.j2 not updated with OOD v3.0 options?

### ISSUE THREAD

**Issue by brandonbiggs** [OSC/ood-ansible#224]
Hi,

It seems like `templates/ondemand.yml.j2` hasn't been updated to reflect all of the new options in OnDemand 3.0 https://osc.github.io/ood-documentation/latest/reference/files/ondemand-d-ymls.html#ondemand-d-ymls

**Comment by johrstrom** [comment]
Yea you're likely right. I can look into this as we prep for 3.1.

**Comment by mrobbert** [comment]
I was just looking for a solution to this problem and came across #257  which solved my issue and I believe it could close this issue.

**Comment by johrstrom** [comment]
> I was just looking for a solution to this problem and came across https://github.com/OSC/ood-ansible/issues/257 which solved my issue and I believe it could close this issue.

Yes thanks for reminding me. I've also documented this feature in the README below.

https://github.com/OSC/ood-ansible?tab=readme-ov-file#ondemandd-configurations
